src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs

```diff
@@ -708,5 +711,63 @@ public async Task ThenTheOrderDescriptionIsDisplayedAsync()
             var orderDescription = order.Description;
             Test.Pages.OrderForm.DeleteConfirmationOrderDescription().Should().BeEquivalentTo(orderDescription);
         }
+
+        [Given(@"the User selects a Catalogue Solution")]
```

Comment:
As this method is no longer used it can be removed

Reply:
done